Source/Core/Common/SymbolDB.cpp

```diff
@@ -39,6 +41,32 @@ void SymbolDB::Index()
 	}
 }
 
+void SymbolDB::PopulateSymbolComments()
+{
+	std::string filename = File::GetUserPath(D_MAPS_IDX) + SConfig::GetInstance().m_strUniqueID + "_comments.txt";
+	if (!File::Exists(filename))
+		return;
+	std::fstream line_parse(filename);
+	std::string line;
+	while (std::getline(line_parse, line))
+	{
+		unsigned int address = 0;
+		if (line.empty())
+			continue;
+		AsciiToHex(line.substr(0,8),address);
```

Comment:
That was a typo; I meant to say there's no sanitizing in the event a line is smaller than 9 characters.